website/content/en/releases/13.3R/readme.adoc

```diff
@@ -91,19 +104,26 @@ There are many sources of information about FreeBSD; some are included with this
 [[release-docs]]
 === Release Documentation
 
-A number of other files provide more specific information about this {releaseType} distribution. These files are provided in various formats. Most distributions will include both ASCII text ([.filename]`.TXT`) and HTML ([.filename]`.HTM`) renditions. Some distributions may also include other formats such as Portable Document Format ([.filename]`.PDF`).
+A number of other files provide more specific information about this {releaseType} distribution.
+These files are provided in various formats.
+Most distributions will include both ASCII text ([.filename]`.TXT`) and HTML ([.filename]`.HTM`) renditions.
```

Comment:
Is it still true that most distributions include all four of `README.TXT`, `RELNOTES.TXT`, `HARDWARE.TXT`, and `ERRATA.TXT` – plus `.HTM` counterparts? 

Without me digging: this smells, to me, like a type of statement that might be outdated. (If my sense of smell is mistaken: I'm probably misguided by <https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264483>.)